instrumentation/netty/netty-4.1/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4_1/AttributeKeys.java

```diff
@@ -26,23 +24,18 @@
             }
           };
 
-  public static final AttributeKey<Context> PARENT_CONNECT_CONTEXT_ATTRIBUTE_KEY =
-      attributeKey("io.opentelemetry.javaagent.instrumentation.netty.v4_1.parent.connect.context");
+  public static final AttributeKey<Context> CONNECT_CONTEXT_ATTRIBUTE_KEY =
```

Comment:
For Netty 4.1, maybe we can just use it's native `AttributeKey.valueOf(Class<?>, String)`

Reply:
👍 I will send shortly in a follow-up PR, I think I've convinced myself we can remove the logic below at the same time, but want to propose in a separate PR for visibility

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/889bce09766d685dd7b3156eee6db46697c8b299/instrumentation/netty/netty-4.1/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4_1/AttributeKeys.java#L47-L53

Reply:
see #1757